src/NodaTime/Calendars/HebrewScripturalCalculator.cs

```diff
@@ -171,17 +171,11 @@ internal static int GetDaysFromStartOfYearToStartOfMonth(int year, int month)
 
         internal static int DaysInMonth(int year, int month) => month switch
         {
-            // FIXME: How do we express multiple cases in a switch expression?
-            // We want: (2, 4, 6, 10, 13) => 29,
-            2 => 29,
-            4 => 29,
-            6 => 29,
-            10 => 29,
-            13 => 29,
+            var m when m == 2 || m == 4 || m == 6 || m == 10 || m == 13 => 29,
```

Comment:
I'd really rather not do this. I think it's less clear, and may well be less efficient too.

Reply:
Well. I'll revert it.
A cleaner way will be added to C# 9.0.